scripts/generate_weight_formats_overview.py

```diff
@@ -14,9 +14,9 @@
 
 # defaults for transition period
 consumer_defaults = {
-    "ilastik": ["pytorch_state_dict", "onnx", "pytorch_script"],
+    "ilastik": ["pytorch_state_dict", "onnx", "torchscript"],
     "zero": ["keras_hdf5"],
-    "deepimagej": ["pytorch_script", "tensorflow_saved_model_bundle"],
+    "deepimagej": ["tensorflow_saved_model_bundle"],
```

Comment:
We shouldn't drop `torchscript` from deepimageJ, we need to support it there in order to demonstrate compatibility with ilastik.
(They may need to update for this one though, but I think that's ok as it shouldn't concern most of the older models.)